corehq/motech/repeaters/management/commands/find_missing_repeat_records.py

```diff
@@ -113,11 +113,7 @@ def find_missing_form_repeat_records_for_form(form, domain, repeaters, enddate,
         if not repeater.allowed_to_forward(form):
             continue
 
-        if repeater.started_at.date() >= enddate:
```

Comment:
We have not carried `started_at` to `SQLRepeater` as it was not used anywhere except for this command. Would like to understand if it's okay to remove the check here or do we want this attribute in the repeaters.
@millerdev @gherceg @kaapstorm

Reply:
Yeah sounds reasonable. If anything, I'd advocate for adding a more "standard" `date_created` and `date_modified`, rather than adding `started_at` back. Here's a relevant slack thread where a few devs weighed in: https://dimagi.slack.com/archives/C02QG63MU/p1643402953907449

Reply:
TLDR: My vote would be only to add `date_created` and `last_modified` when you need them. But if you do need them, I agree, don't call it `started_at`.

Ancient history: `started_at` could definitely have been named better. It was meant to be "started or last unpaused at". The intention was to figure out when a remote endpoint was offline, or no longer existed, so we could skip/drop repeat records that we are certain would fail.

In the end we backed out. Avoiding resource conflicts on the Repeater was hard, and we figured we'd just wait until we had SQL repeaters.

Reply:
P.S. Based on that Slack thread, it looks like we do have a use case for `date_created` & `last_modified` ("... it looks like a saved model has a data value that is no longer supported. I’d like to know when that data value was initially inserted ...") so, sure, let's add them.

Reply:
Thank you everyone for your valuable input. They are of great help.
To conclude- 
- I will add `date_created` and `last_modified` on the `SQLRepeater` model.
- I will also add `started_at`, `last_success_at`, and `failure_streak` attributes and will put up a PR afterward on how we plan to use them. It seems they will add value as described by Norman.

After understanding the use case, I see that `started_at` variable is making sense. I think adding a comment would be helpful. Another thought that is coming to my mind is adding `paused_at` and `unpaused_at` in the `SQLRepeater`. @kaapstorm What do you think which approach would be better?

Reply:
I still think it could be useful to automatically throttle or pause repeaters that appear to be offline. But, personally, I'd wait before adding the fields until we have a spec or a plan or a mental model for exactly how we want to do that with the SQL models.